tools_galaxyp.yaml

```diff
@@ -1,158 +1,925 @@
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_plugin_fixploidy
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_reheader
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_plugin_tag2tag
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_annotate
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_stats
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_mpileup
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_query
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
 
-  - name: maldi_quant_preprocessing
+  - name: bcftools_consensus
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_gtcheck
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_plugin_fill_an_ac
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: bcftools_plugin_vcf2sex
+    owner: iuc
+    tool_panel_section_label: Bcf Tools 
+
+  - name: vcfdistance
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: blastxml_to_tabular_selectable
     owner: galaxyp
-    tool_panel_section_label: Proteomics
+    tool_panel_section_label: Blast
+
+  - name: variant_select
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: print_reads
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: table_recalibration
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variant_apply_recalibration
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variant_annotator
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variant_eval
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: depth_of_coverage
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variants_validate
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: realigner_target_creator
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: count_covariates
+    owner: devteam
+    tool_panel_section_label: NGS
 
-  - name: maldi_quant_peak_detection
+  - name: variant_combine
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: unified_genotyper
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: analyze_covariates
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: indel_realigner
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variant_filtration
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: variant_recalibrator
+    owner: devteam
+    tool_panel_section_label: NGS
+
+  - name: directag_and_tagrecon
     owner: galaxyp
-    tool_panel_section_label: Proteomics
+    tool_panel_section_label: Misc
+
+  - name: blosum
+    owner: oscar_b
+    tool_panel_section_label: Beta Test Tools
+
+  - name: select_random_lines
```

Comment:
there is a tool already that does exactly this: https://usegalaxy.eu/root?tool_id=random_lines1